### PR TITLE
Jetpack pricing: add open-source section

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR adds a section mentioning open-source to the Jetpack pricing page.

Context: document linked in p1HpG7-j1R-p2

### Testing instructions

- Spin-up the pricing page in Cloud, using the live link below, or by running this branch locally
- Check that the open-source section looks like the capture below
- Do the same with Calypso blue

### Screenshots

_Cloud_
Screen Shot 2022-11-15 at 10 19 48 AM

_Calypso blue_
Screen Shot 2022-11-15 at 10 20 14 AM

### CODE REVIEW

File: client/my-sites/plans/jetpack-plans/open-source/style.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+.jetpack-open-source__icon {
+	margin-bottom: 1.5rem;
+}
+
+.jetpack-open-source__title {
+	margin-bottom: 1rem;
+
+	color: var(--color-text);
+	font-size: 1.5rem !important;
```
Comment (monsieur-z) [inline client/my-sites/plans/jetpack-plans/open-source/style.scss:25]: Not ideal, but `h2` in Calypso blue have a high specificity.

File: client/my-sites/plans/jetpack-plans/product-store/need-more-info/style.scss [hunk 2]
```diff
@@ -2,7 +2,6 @@
 @import "@wordpress/base-styles/mixins";
 
 .jetpack-product-store__need-more-info {
-	background-color: var(--color-surface);
```
Comment (monsieur-z) [inline client/my-sites/plans/jetpack-plans/product-store/need-more-info/style.scss:5]: A white background looked odd in Calypso blue.

**Review by keoshi — CHANGES_REQUESTED** [review]
Hey, @monsieur-z — this is looking good both on desktop and mobile!

The only notes I have are in relation to the typography in the title. Could we maybe approximate the weight and size to the “Need more info?” heading so that this section is more consistent with the rest of the page?

Before | After
--|--
Calypso Blue
image | image
Calypso Green
image | image

Thanks for doing this!

**Comment by monsieur-z** [comment]
> Hey, @monsieur-z — this is looking good both on desktop and mobile!
> 
> The only notes I have are in relation to the typography in the title. Could we maybe approximate the weight and size to the “Need more info?” heading so that this section is more consistent with the rest of the page?

Sure thing!

**Review by elliottprogrammer — APPROVED** [review]
Tested in Calypso blue and green. Everything looks all good to me. Code looks good, I don't see any issues.
LGTM! 👍

**Review by keoshi — APPROVED** [review]
Looks great to me! 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7626331

Thank you @monsieur-z for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7626331).

**Pull request merged**